task_DEV_3/task_DEV_3/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+
+namespace task_DEV_3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter the length of the array :");
+            int arrayLength = Convert.ToInt32(Console.ReadLine());
+
+            if (arrayLength <= 5)
+            {
+                int[] numberArray = new int[arrayLength];
+
+                for (int i = 0; i < arrayLength; i++)
+                {
+                    Console.WriteLine($"Enter an array element № {i}");
+                    numberArray[i] = Convert.ToInt32(Console.ReadLine());
+                }
+                bool isPrimeNumber = true;
```

Comment:
It would be better to swap 21 and 22 lines just for readability, Because isPrimeNumber is uesd in the second loop.

Reply:
And in such case we can to not initialize isPrimeNumber here because it is initialized in the loop. We can leave just `bool isPrimeNumber`.